### ISSUE TITLE
Form validation: "required" spoken twice for screen readers 

### ISSUE THREAD

**Issue by SBleeks** [wet-boew/wet-boew#7002]
Why is the word "required" identified and spoken twice?  For example using JAWS 16 and/or NVDA (latest build), the word "required" is spoken by the screen reader before and after the required edit field.
@wet-boew/access-working-group

**Comment by dfarough** [comment]
I am thinking that both Aria-required and the html 5 required attributes are being used for these fields.  Unless there is a good reason to use aria-required I think that it should be removed in favour of the html 5 required attribute.

**Comment by SBleeks** [comment]
Agreed.  That makes sense to me.  Could we test this approach?

**Comment by nschonni** [comment]
Not sure if this issue should be merged with your previously reported on in #6966, but I'll answer below
____

I'm not sure if the double required is coming from the aria-required, which gets programmatically added for compatibility with old IE that doesn't understand HTML.

The other cause may be because the demo label includes the text "required". See https://github.com/wet-boew/wet-boew/blob/master/src/plugins/formvalid/formvalid-en.hbs#L26
```html
				<div class="form-group">
					<label for="title1" class="required"><span class="field-name">Title</span> <strong class="required">(required)</strong></label>
					<select class="form-control" id="title1" name="title1" required="required">
						<option label="Select a title"></option>
						<option value="dr">Dr.</option>
						<option value="esq">Esq.</option>
						<option value="mr">Mr.</option>
						<option value="ms">Ms.</option>
					</select>
				</div>
```

Would a plain page with and without aria-required help trace the issue?

**Comment by dfarough** [comment]
I wondered whether the label might be the issue.

Pages with and without aria-required sounds like a good idea.

**Comment by pjackson28** [comment]
It definitely is the label as otherwise you would hear required three times. We need it in the label as well since both required and aria-required are of little to no use to visual users (e.g., screen magnification users) so I don't see how the duplication can be avoided.

**Comment by hsrudnicki** [comment]
Please update the status of this issue whether this issue can be closed or otherwise?

**Comment by hsrudnicki** [comment]
Can be this issue closed or otherwise, pls update its status

**Comment by hsrudnicki** [comment]
Also, as in the referenced issue (6966), made this issue as "Going Stale" to close or if otherwise?
Either way, please update this issue's status whether this issue can be closed or otherwise?
Thank you

**Comment by dfarough** [comment]
I agree that having required in the label is the best way to handle this issue.  Often space is a concern so an asterisk is used instead.  I think here we are testing the messaging and the form validation functionality, so if we are including this in the label maybe we don't need it on the field as well. I tried changing verbosity levels to see if I could eliminate any of this duplication.  this was not possible.

**Comment by LaurentGoderre** [comment]
I don't think we should be removing it from the field because this is not likely future-proof. Fuiture versions of HTML5 or browser implementation could put the attribute to good use.

**Comment by dorina1** [comment]
@wet-boew/access-working-group 
Because we use required="required" in the form elements tags we hear required twice: once from the label, and second time once we are in the form element. What we find problematic is not only that we hear it twice, but on the French pages we hear first obligatoire and then required (once in French then in English). Also the asterisk at the beginning of the label comes from CSS, so JAWS completely skip that. JAWS users need to listen to the whole label in order to hear at the end required. If the asterisk would be hard coded (not added by CSS) screen reader users could have the same experience as everybody else and pick the required fields faster. The * is not an accessibility issue but a user experience improvement. Can the  required="required"  be removed so we will hear only once required?

**Comment by hsrudnicki** [comment]
Please update this issue's status whether this issue can be closed as resolved/fixed or otherwise?

**Comment by hsrudnicki** [comment]
Again, if this remains an issue that is duplicated in the issue referenced 7540, can be this closed or otherwise?

**Comment by SBleeks** [comment]
Yes, this issue can be closed.
Thank you.



Stacy Bleeks

Web Accessibility Expert - Centre of Expertise on Accessibility | Communications and Dissemination Branch
Statistics Canada | Government of Canada
Stacy.Bleeks@canada.ca<mailto:Stacy.Bleeks@canada.ca> / Tel: 613-951-6137

Expert en accessibilité  - Centre d'expertise en accessibilité | Direction des communications et de la diffusion
Statistique Canada | Gouvernement du Canada
Stacy.Bleeks@canada.ca<mailto:Stacy.Bleeks@canada.ca> / Tél : 613-951-6137

**Comment by hsrudnicki** [comment]
Based on the above, this issue can be closed, thank you
